apiserver/facades/client/bundle/bundle.go

```diff
@@ -151,10 +156,111 @@ func (b *BundleAPI) GetChanges(args params.BundleChangesParams) (params.BundleCh
 }
 
 // ExportBundle exports the current model configuration as bundle.
-func (b *BundleAPI) ExportBundle() (params.StringResult, error) {
-	return params.StringResult{}, nil
+func (b *BundleAPI) ExportBundle() (params.BytesResult, error) {
```

Comment:
We cannot just remove 
```
ExportBundle() (params.StringResult, error)
```
as v1 should still have it to comply with the original promise.

Also why would we return a BytesResult? or even a StringResult? We customary return map[string]interface{} for similar situations and let the clients parse the results. For example, see modelmanager.DumpAll()

Reply:
@anastasiamac I asked @vinu2003 to do it this way as this is the new correct way to implement versioned APIs. 

I agree that it should be a `StringResult` rather than a `BytesResult` as it will be easier for the gui to consume.

@anastasiamac for this call we are wanting to return the bundle, which is a YAML document, not a map[string]interface{}.

Reply:
I see! Ok, it makes sense then to marshal it here and return as params.StringResult.

Reply:
@howbazaar actually mentioned this in other PR.  that no ther format is accpeted for bundle. the confusion started when i changed it to BytesResult. Apologise. I never considered that gui takes this value.